### PR TITLE
[aptos-cli] Fix validator config and validator set CLI commands

### PR DESCRIPTION
**Pull request by gregnazario**
### Description
Before it didn't print network addresses, now it prints network addresses properly

### Test Plan
```
aptos node show-validator-set                
{
  "Result": {
    "scheme": "BLS12381",
    "active_validators": [
      {
        "account_address": "c2365fd303b4473729a59d02f60f3747e8dd7ecdb25351fde1e52865e8d81a85",
        "consensus_voting_power": 100015982794673,
        "config": {
          "consensus_public_key": "0xb7b923559545c113d06a4ff29c258e595d740191e21be0fcb067eb25635d6a455f1544caaae38e7532f93201b9add6b7",
          "validator_network_addresses": [
            "/dns/dev-aptos-node-24-validator-lb/tcp/6180/noise-ik/0xabd5f52d87cfb33689abb17e6cbf2a41964943f0d85d81fac75ebdda36b35638/handshake/0"
          ],
          "fullnode_network_addresses": [
            "/dns/fullnode24.devnet.aptoslabs.com/tcp/6182/noise-ik/0x5fa7dc900f85384ff4133b00f6056da93b6ccd9981514fe33a648e2468189f65/handshake/0"
          ],
          "validator_index": 0
        }
      },
      {
        "account_address": "973cef624dcad82fe952a784d696afe14e9f7b53115955297577d42b69926725",
        "consensus_voting_power": 100015982794673,
        "config": {
          "consensus_public_key": "0x824a68c25e2b179196f5dc54374200e74d1a510164a481ad1641515d31b0ca765b973127ebd828318fbf946f6533c371",
          "validator_network_addresses": [
            "/dns/dev-aptos-node-23-validator-lb/tcp/6180/noise-ik/0xfd79be48a0378f569b11bb4c3d9388d959d598cb379b11d7596c813757871631/handshake/0"
          ],
          "fullnode_network_addresses": [
            "/dns/fullnode23.devnet.aptoslabs.com/tcp/6182/noise-ik/0xb7f954ff9a2edd8acdc644b598581ba9d641025accf80851701341d3aa758226/handshake/0"
          ],
          "validator_index": 1
        }
      },
      {
        "account_address": "16e161b05cdff097f415140d2e418bb82f4673a41602d7bf650db07974aa640e",
        "consensus_voting_power": 100015982794673,
        "config": {
          "consensus_public_key": "0x8606b66b3fe6649a1da69188483a2a580794322ae7537bb57ae531519898ec44d6a1b83894035d5fca163ee5a4ebfdb2",
          "validator_network_addresses": [
            "/dns/dev-aptos-node-22-validator-lb/tcp/6180/noise-ik/0x907316ebb5d11c676a82a7245f785e291a08ab93cbe72dbed071d236f064c90b/handshake/0"
          ],
          "fullnode_network_addresses": [
            "/dns/fullnode22.devnet.aptoslabs.com/tcp/6182/noise-ik/0x2470b25f252c9b03a3e0ab8b3ef9c487a7c351bfffe4b52e56ec363bd744fe76/handshake/0"
          ],
          "validator_index": 2
        }
      },
      {
        "account_address": "4af96ce323e41e54e8cd760a53920fe975b0bb37c1a7e6c71f586c4a70f86810",
        "consensus_voting_power": 100015982794673,
        "config": {
          "consensus_public_key": "0xaf66097ac6d9b3ea60f366328646303c3286581098dd85a7009e626a899b71bd43cda78f2e93f489dbfe5a7b49738be3",
          "validator_network_addresses": [
            "/dns/dev-aptos-node-21-validator-lb/tcp/6180/noise-ik/0x07bf2bb3a8f51e4ad227e57718ceba6eced7e474c72354b94d5d6514aadb720f/handshake/0"
          ],
          "fullnode_network_addresses": [
            "/dns/fullnode21.devnet.aptoslabs.com/tcp/6182/noise-ik/0x4045eef09e63d8ecd74856678965747d6d1565a7dd16bdd4e52ed56315903831/handshake/0"
          ],
          "validator_index": 3
        }
      },

```

```
aptos node show-validator-config --pool-address d488a9ed4243198dff9d53c364a4ff309c715caa7915db31ae258d3a158b21bf
{
  "Result": {
    "consensus_public_key": "0x8ea105d3d18c65322c0191b9006c3bb0b8ce2185fd75f778d2efc318b1e50290c356367b84bd0a63e8f114405186e792",
    "validator_network_addresses": [
      "/dns/dev-aptos-node-0-validator-lb/tcp/6180/noise-ik/0x4265ce2aa01b2741028c364594195f7d26f9432915946c655e10f3bc8bbdf645/handshake/0"
    ],
    "fullnode_network_addresses": [
      "/dns/fullnode0.devnet.aptoslabs.com/tcp/6182/noise-ik/0x28f84d49d44754fa381fe807ef6e94816a36aa5600ec4018fba0fa8df9fe2f4d/handshake/0"
    ],
    "validator_index": 24
  }
}

```

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/4054)

### CODE REVIEW

File: crates/aptos/src/node/mod.rs [hunk 1]
```diff
@@ -447,17 +455,152 @@ pub struct ShowValidatorSet {
...
+            .await?
+            .into_inner();
+
+        ValidatorSetSummary::try_from(&validator_set)
+            .map_err(|err| CliError::BCS("Validator Set", err))
+    }
+}
+
+#[derive(Clone, Debug, Eq, PartialEq, Serialize)]
+pub struct ValidatorSetSummary {
```
Comment (zekun000) [inline crates/aptos/src/node/mod.rs:476]: we have the same struct here https://github.com/aptos-labs/aptos-core/blob/7c0b9fc366d8ac4596d6f07d9b42630586b9be8d/types/src/on_chain_config/validator_set.rs#L25

is there a reason you want to re-implement them?

**Review by jjleng — APPROVED** [review]
unblock here. but what's the reason for replacing `serde_json::Value`?

File: crates/aptos/src/node/mod.rs (hunk 1 above)
Comment (gregnazario) [reply crates/aptos/src/node/mod.rs:476]: Those structs don't show the network addresses.  This PR converts them to readable versions

**Comment by gregnazario** [comment]
> unblock here. but what's the reason for replacing `serde_json::Value`?

It doesn't show the network addresses.  They show BCS encoded

**Review by sherry-x — APPROVED** [review]

**Pull request merged**